docs/source/usersguide/input.rst

```diff
@@ -1110,13 +1110,14 @@ Each ``material`` element can have the following attributes or sub-elements:
 
     *Default*: ""
 
-  :density:
-    An element with attributes/sub-elements called ``value`` and ``units``. The
-    ``value`` attribute is the numeric value of the density while the ``units``
-    can be "g/cm3", "kg/m3", "atom/b-cm", "atom/cm3", or "sum". The "sum" unit
-    indicates that the density should be calculated as the sum of the atom
-    fractions for each nuclide in the material. This should not be used in
-    conjunction with weight percents.
+  :density: An element with attributes/sub-elements called ``value`` and
```

Comment:
For the most part, the `.rst` files look to be organized with the attributes on one line and the start of the statement indented two spaces on the next line. I would recommend moving the start of the statement to the next line to keep the syntax consistent here.